src/html-conventions-lib/info-box.xsl

```diff
@@ -0,0 +1,176 @@
+                            </xsl:call-template>
+                        </ul>
+                    </div>
+                </div>
+            </li>
+        </xsl:for-each>
+    </xsl:template>
+
+
+    <xsl:template name="moreDetailsForAttributes">
+        <xsl:param name="attributeName"/>
+        <xsl:param name="root"/>
+        <xsl:variable name="attributesWithSameName"
+            select="f:getClassAttributeByName($attributeName, $root)"/>
+        <xsl:if test="fn:count($attributesWithSameName) > 1">
+            <xsl:for-each select="$attributesWithSameName">
+                <xsl:variable name="className" select="./parent::attributes/parent::element/@name"/>
+                <xsl:variable name="attributeType" select="./properties/@type"/>
+                <li>
+                    <xsl:value-of select="fn:concat($className, ' (', $attributeType, ')')"/>
+                </li>
+            </xsl:for-each>
+        </xsl:if>
+    </xsl:template>
+
+
+    <xsl:template name="listOfConnectorsAndOccurrences">
+        <xsl:variable name="root" select="root()"/>
+        <xsl:variable name="connectorNames" select="f:getDistinctConnectorsNames($root)"/>
+        <xsl:for-each select="$connectorNames">
+            <xsl:variable name="occurences"
+                select="
+                    if (fn:count((f:getConnectorByName(., $root))) > 1) then
+                        fn:concat(' (', fn:string(fn:count((f:getConnectorByName(., $root)))), ')')
+                    else
+                        ()"/>
+            <xsl:variable name="connectorName" select="."/>
+            <xsl:variable name="hrefConnectorName"
+                select="
+                    if (fn:contains($connectorName, ':')) then
+                        f:camelCaseString(fn:substring-after($connectorName, ':'))
+                    else
+                        f:camelCaseString($connectorName)"/>
+            <li class="list-group-item">
+                <a class="btn btn-link" data-toggle="collapse" href="#collapse{$hrefConnectorName}"
+                    role="button" aria-expanded="false" aria-controls="collapse{$hrefConnectorName}">
+                    <xsl:value-of select="$connectorName"/>
+                    <xsl:value-of select="$occurences"/>
+                </a>
+                <div class="collapse" id="collapse{$hrefConnectorName}">
+                    <div class="card card-body">
+                        <ul>
+                            <xsl:call-template name="moreDetailsForConnectors">
+                                <xsl:with-param name="connectorName" select="$connectorName"/>
+                                <xsl:with-param name="root" select="$root"/>
+                            </xsl:call-template>
+                        </ul>
+                    </div>
+                </div>
+            </li>
+        </xsl:for-each>
+    </xsl:template>
+
+    <xsl:template name="moreDetailsForConnectors">
```

Comment:
more? are there also less of them? perhaps a better name would be more suggestive.

Reply:
renamed